### PR TITLE
Feature/user stories display pantry

### PR DESCRIPTION
**Pull request by Alynn022**
PR Template:

Is this a fix or a feature?
- [ ] Bug fix
- [x] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor: Add functionality for user to be able to add recipe to cook when they have enough ingredients

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test: scripts.js

Additional comments:

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -377,23 +396,25 @@ const displaySelectedFavoriteRecipe = (event) => {
...
 };
 
-const userPantryName = document.getElementById('userPantryName')
-
 const displayUserPantry = () => {
-  // newUser = newUser(userData[0]);
-  let userPantryIngredients = newUser.showPantryIngredientInfo();
+  let userPantryIngredients = pantry.showPantryIngredientInfo();
   userPantryName.innerText = `${newUser.name}'s Pantry`
+  if (userPantryIngredients.length === 0) {
```
Comment (StevieAmb) [inline src/scripts.js:406]: I love this edge casing.

**Review by lschnell8 — APPROVED** [review]
Nice teamwork on this ya'll!

File: src/index.html [hunk 2]
```diff
@@ -48,21 +48,25 @@ <h2>Recipe Title</h2>
...
     </ul>
+    <h3 class="total-cost">Recipe Cost Total: <span id="totalCost">$$$$$</span></h3>
     <h3>Recipe Instructions</h3>
     <ol id="instructionsList">
     </ol>
-    <h3 class="total-cost">Recipe Cost Total: <span id="totalCost">$$$$$</span></h3>
     <aside class="user-pantry-view" id="userPantryView">
       <h3 class="user-pantry-name" id="userPantryName"></h3>
     </aside>
+    <section class="ingredients-needed-for-user" id="ingredientsNeededForUser">
```
Comment (StevieAmb) [inline src/index.html:64]: It's awesome to see this method be displayed.

File: src/scripts.js [hunk 3]
```diff
@@ -102,9 +107,11 @@ const loadPage = () => {
     newUser = new User(randomUser);
     cookbook = new RecipeBox(data[1]);
     recipe = new Recipe(data[1]);
+    pantry = new Pantry(newUser)
     groceryStore = data[0];
     displayHomePage();
   })
+  console.log('here', recipe)
```
Comment (StevieAmb) [inline src/scripts.js:114]: We gotta remember to get rid of this console.log()! Awesome way to check our stuff, though. Genius!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**